src/components/Button/StrokeButton.mdx

```diff
@@ -0,0 +1,82 @@
+---
+name: StrokeButton
+menu: Reusable Components
+---
+
+import { Playground, Props } from 'docz'
+import { Toggle as ToggleState } from 'react-powerplug'
+
+import { COLORS } from '../../constants'
+import StrokeButton from './StrokeButton'
+
+# StrokeButton
+
+<Props of={StrokeButton} />
+
+
+## Basic usage
+
+<Playground>
+  <StrokeButton>Default</StrokeButton>
+</Playground>
+
+
+## Sizes
+
+<Playground>
+  <StrokeButton size="xsmall">
+    Extra-small button
+  </StrokeButton>
+</Playground>
+
+<Playground>
+  <StrokeButton size="small">
+    Small button
+  </StrokeButton>
+</Playground>
+
+<Playground>
+  <StrokeButton size="medium">
+    Medium button
+  </StrokeButton>
+</Playground>
+
+<Playground>
+  <StrokeButton size="large">
+    Large button
+  </StrokeButton>
+</Playground>
+
+## Colors
+
+<Playground>
+  <StrokeButton color1={COLORS.green[500]} color2={COLORS.green[500]}>
```

Comment:
Looks like there was maybe an api change, color1 and color2 in this file need to be replaced with strokeColors, e.g:

```js
<Playground>
  <StrokeButton strokeColors={[COLORS.green[500], COLORS.green[500]]}>
    Solid green
  </StrokeButton>
</Playground>
```